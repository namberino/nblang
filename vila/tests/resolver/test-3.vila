fun test()
{
    mut t = "1";
    mut t = "2"; // error
}
