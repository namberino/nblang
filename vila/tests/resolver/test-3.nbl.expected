On line: 4, Error at 't': Already a variable with this name in this scope
